plugins/iexec.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: iexec
+spec:
+  version: "0.1.9"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [linux]}
+    uri: https://github.com/gabeduke/kubectl-iexec/releases/download/0.1.14/kubectl-iexec_0.1.14_Linux_x86_64.tar.gz
+    sha256: "8d69b66309d31a5422ea742398751d9b9eacc2f4bd55b8673cb3f9e6bbbb4af9"
+    files:
+    - from: "/kubectl-iexec"
+      to: "."
+    bin: "./kubectl-iexec"
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin]}
+    uri: https://github.com/gabeduke/kubectl-iexec/releases/download/0.1.14/kubectl-iexec_0.1.14_Darwin_x86_64.tar.gz
+    sha256: "e1c5524384fa5db09a84341ebd52bcfaa9d132fa341889c60ccb067335781d77"
+    files:
+    - from: "/kubectl-iexec"
+      to: "."
+    bin: "./kubectl-iexec"
+  shortDescription: Interactive selection tool for `kubectl exec`
+  homepage: https://github.com/gabeduke/kubectl-iexec
+  description: |
+    _          _               _   _        _
```

Comment:
you might want to show your ascii art in `caveats` field, along with some examples perhaps.

right now I'm not very positive about `description` field having monospace art, as we might render them on readme, or an html site perhaps. `caveats` field is shown in the terminal after installing the plugin, so it's more appropriate. see other plugins as a reference.

if I were you, I'd remove ascii art altogether for the sake of simplicity. it doesn't add much to the experience.

Reply:
Removed as suggested. Fixed in the upstream tool help menu as well.